code/backend/src/main/resources/application.yaml

```diff
@@ -0,0 +1,80 @@
+spring:
```

Comment:
I usually split the application.yaml into separate files for dev, prod etc. You can simply put the profile name as the suffix in the name (`application-dev.yaml`), and spring will automatically load the corresponding file(s) based on the currently active profiles.